lightning/src/onion_message/messenger.rs

```diff
@@ -357,6 +371,103 @@ where
 		}
 	}
 
+	/// Decode one layer of an incoming onion message
+	/// Returns either a Forward (another onion message), or Receive (decrypted content)
+	pub fn peel_onion<T: CustomOnionMessageContents>(
+		node_signer: &NS,
+		secp_ctx: &Secp256k1<secp256k1::All>,
+		logger: &L, 
+		custom_handler: &CMH,
+		msg: &msgs::OnionMessage,
```

Comment:
nit: 
```suggestion
		node_signer: &NS, secp_ctx: &Secp256k1<secp256k1::All>, logger: &L, custom_handler: &CMH,
		msg: &msgs::OnionMessage,
```

We don't follow the `rustfmt` guideline of one parameter per line, similar below when calling `peel_onion` (just keep to 100 chars per line)